cmd/version/version.go

```diff
@@ -117,10 +117,14 @@ func checkForLatestVersion(currVersion string) {
 		return
 	}
 
+	
 	latestVersion := data["spec"].(map[string]interface{})["version"].(string)
 	if !isLatestVersion(currVersion, latestVersion) {
-		// The seperator for the error print
 		fmt.Println()
+		if currVersion == "dev" {
+			fmt.Printf("You are using development version of cli, latest released version is: %s", latestVersion)
```

Comment:
```suggestion
			fmt.Println("You are using development version of cli, latest released version is: " + latestVersion)
```

Reply:
> Println is giving warning so, I replaced this with Printf

That's because Println had `%s`, as it was a format specifier or a placeholder for a string, this one shouldn't give warnings.